packages/react-i18n/src/test/manager.test.ts

```diff
@@ -524,7 +524,7 @@ function createTranslationPromise(
     };
   });
 
-  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore (It thinks that the resolve/ reject are not assigned yet,
```

Comment:
use `@ts-expect-error It thinks that the resolve/ reject are not assigned yet` and you shouldn't need the eslint-disable